src/constructs/boxes.md

```diff
@@ -2,7 +2,33 @@
 title: Boxes
 ---
 
-Value bindings in Grain are immutable, which means that they do not change after they have been declared. While there's a ton we can get done with regular bindings, it can sometimes be helpful to change the data already associated with a name. This is accomplished with what we call "boxes."
+## On the subject of Boxes
+
+Boxes can be used for adding a mutable property to an immutable data structure, like a tuple or list, but `let mut` should be preferred for creating mutable value bindings.
```

Comment:
I've been wanting to get some banners in here... This would be a great time to have an info banner for the second sentence.
```suggestion
Boxes can be used for adding a mutable property to an immutable data structure, like a tuple or a list. `let mut` should be preferred when creating mutable value bindings.
```